lib/config.js

```diff
@@ -35,6 +36,20 @@ exports.get = () => {
   if (errors.length > 0) {
     throw new Error('Error in .tldrrc configuration:\n' + errors.join('\n'));
   }
+
+  // Setting correct languages
+  merged['languages'] = ['en'];
```

Comment:
Using similar conventions as above, better to do `merged.languages` vs `merged['languages']`.